Packs/Netmiko/Integrations/Netmiko/Netmiko.py

```diff
@@ -126,25 +134,38 @@ def cmds_command(client, args):
             demisto.error(f"Error with raw print output - {err}")
 
     else:
-        md = tableToMarkdown(f'Command(s) against {client.hostname} ({client.platform}):', output.get('Commands', []))
+        hdrs = ["Hostname", "DateTimeUTC", "Command", "Output"]
+        data = []
+
+        # Single command
+        if len(cmds) == 1:
+            data.append(output["Commands"][0])
```

Comment:
is `Commands` key guaranteed to exist under `outputs`?

Reply:
Yes, Commands it the output returned after the cmds= commands are executed. In this case, only a single command was passed as an argument (ifconfig, whoami, etc.).